src/controllers/__tests__/authenticationController.test.ts

```diff
@@ -882,6 +893,60 @@ describe("AuthenticationController#acs", () => {
     expect(mockGetProfile).not.toBeCalled();
     expect(mockCreateProfile).not.toBeCalled();
   });
+
+  it.each`
+    inputFc              | FF                        | TEST_USERS       | expectedResult
+    ${aFiscalCode}       | ${FeatureFlagEnum.NONE}   | ${[]}            | ${false}
+    ${aFiscalCode}       | ${FeatureFlagEnum.BETA}   | ${[aFiscalCode]} | ${true}
```

Comment:
use directly the expected value
```suggestion
    inputFc              | FF                        | TEST_USERS       | expectedUriScheme
    ${aFiscalCode}       | ${FeatureFlagEnum.NONE}   | ${[]}            | ${"https:"}
    ${aFiscalCode}       | ${FeatureFlagEnum.BETA}   | ${[aFiscalCode]} | ${"iologin:"}
```

Reply:
refactored this test in https://github.com/pagopa/io-backend/pull/1012/commits/3ada102f061d73a9a4fc9258f840af9f6d3e6b7b